fastlane/metadata/i18ncli.js

```diff
@@ -0,0 +1,320 @@
+      console.log( "Removed existing directory", newDirectoryPath );
+    } catch ( e ) {
+      // Directory did not exist
+    }
+    console.log( "Renaming directory", directoryPath, "to", newDirectoryPath );
+    await fsp.rename( directoryPath, newDirectoryPath );
+  } ) );
+}
+
+async function removeUnsupportedDirectories( ) {
+  // Get all directories in fastlane/metadata/ios
+  const iosDirectories = await fsp.readdir( path.join( __dirname, "ios" ), {
+    withFileTypes: true
+  } ).then( files => files.filter( file => file.isDirectory( ) ).map( file => file.name ) );
+  console.log( "Current list of the directories in fastlane/metadata/ios", iosDirectories );
+  await Promise.all( iosDirectories.map( async directory => {
+    if ( SUPPORTED_IOS_METADATA_LOCALES.indexOf( directory ) >= 0 ) return;
+    const directoryPath = path.join( __dirname, "ios", directory );
+    console.log( "Removing unsupported directory", directoryPath );
+    await fsp.rmdir( directoryPath, { recursive: true } );
+  } ) );
+  const androidDirectories = await fsp.readdir( path.join( __dirname, "android" ), {
+    withFileTypes: true
+  } ).then( files => files.filter( file => file.isDirectory( ) ).map( file => file.name ) );
+  console.log( "Current list of the directories in fastlane/metadata/android", androidDirectories );
+  return Promise.all( androidDirectories.map( async directory => {
+    if ( SUPPORTED_ANDROID_METADATA_LOCALES.indexOf( directory ) >= 0 ) {
+      console.log( "Supported directory", directory );
+      return;
+    }
+    const directoryPath = path.join( __dirname, "android", directory );
+    console.log( "Removing unsupported directory", directoryPath );
+    await fsp.rmdir( directoryPath, { recursive: true } );
+  } ) );
+}
+
+async function copyAndroidTitle() {
+  // Get all directories in fastlane/metadata/android
+  const androidDirectories = await fsp
+    .readdir( path.join( __dirname, "android" ), {
+      withFileTypes: true
+    } )
+    .then( files => files.filter( file => file.isDirectory() ).map( file => file.name ) );
+  // Copy title.txt from en-US directory into all the other directories
+  const titlePath = path.join( __dirname, "android", "en-US", "title.txt" );
+  await Promise.all(
+    androidDirectories.map( async directory => {
+      const directoryPath = path.join( __dirname, "android", directory );
+      console.log( "Copying title.tx into: ", directoryPath );
+      await fsp.copyFile( titlePath, path.join( directoryPath, "title.txt" ) );
+    } )
+  );
+}
+
+// eslint-disable-next-line no-unused-expressions
+yargs
+  .usage( "Usage: $0 <cmd> [args]" )
+  .command(
+    "build",
+    "Prepare existing localizations for uploading metadata",
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    ( ) => {},
+    async ( ) => {
+      await renameDirectories();
+      await removeUnsupportedDirectories();
+      await copyAndroidTitle();
```

Comment:
IMO it might be helpful to print status updates at this level, e.g.

```
Renaming directories...
Removing unsupported directories...
Copying Android title...
```

Reply:
Added these logs